tasks/deploy/config/linea/factory.ts

```diff
@@ -0,0 +1,74 @@
+import { Address, parseEther, zeroAddress } from "viem";
+import { HyperdriveFactoryConfig, parseDuration } from "../../lib";
+
+// The name of the factory.
+export const LINEA_FACTORY_NAME = "ElementDAO Hyperdrive Factory";
+
+// The name of the forwarder factory.
+export const LINEA_FACTORY_FORWARDER_NAME =
+    "ElementDAO ERC20 Factory Forwarder";
+
+export const LINEA_FACTORY: HyperdriveFactoryConfig = {
+    name: LINEA_FACTORY_NAME,
+    prepare: async (hre, options) => {
+        await hre.hyperdriveDeploy.ensureDeployed(
+            LINEA_FACTORY_FORWARDER_NAME,
+            "ERC20ForwarderFactory",
+            [LINEA_FACTORY_FORWARDER_NAME],
+            options,
+        );
+    },
+    constructorArguments: async (hre) => [
+        {
+            governance: (await hre.getNamedAccounts())["deployer"] as Address,
+            deployerCoordinatorManager: (await hre.getNamedAccounts())[
+                "deployer"
+            ] as Address,
+            hyperdriveGovernance: (await hre.getNamedAccounts())[
+                "deployer"
+            ] as Address,
+            defaultPausers: [
+                (await hre.getNamedAccounts())["deployer"] as Address,
+                (await hre.getNamedAccounts())["pauser"] as Address,
+            ],
+            feeCollector: zeroAddress,
+            sweepCollector: zeroAddress,
+            checkpointRewarder: zeroAddress,
+            checkpointDurationResolution: parseDuration("1 hours"),
+            minCheckpointDuration: parseDuration("24 hours"),
+            maxCheckpointDuration: parseDuration("24 hours"),
+            minPositionDuration: parseDuration("7 days"),
+            maxPositionDuration: parseDuration("730 days"),
+            minFixedAPR: parseEther("0.005"),
+            maxFixedAPR: parseEther("0.1"),
```

Comment:
why is this only 10%? we've seen ezeth higher, right?

Reply:
I think I copied this from the other factory config. I will take a look when I’m back.

Reply:
okay cool, we might wanna change the min/max ts config value too

Reply:
We talked about this offline. We're going to leave this as is since this is protecting deployers from initializing a pool in such a way that the pool can't be traded close to a fixed rate of 0%.